message_reader.cpp

```diff
@@ -101,7 +101,8 @@ void TelegramRecorder::markMessageAsRead(std::shared_ptr<td_api::message>& messa
   SPDLOG_DEBUG("Marking message {} from chat {} as read", message->id_, message->chat_id_);
   td_api::object_ptr<td::td_api::viewMessages> viewMessages = td_api::make_object<td_api::viewMessages>();
   viewMessages->chat_id_ = message->chat_id_;
-  viewMessages->message_thread_id_ = message->message_thread_id_;
```

Comment:
Why are we commenting this line?

Reply:
New API doesn't have such field, unless you do static cast to certain specific type. There may be several specific types and new ones in the future versions too.
Mark as read request  works even without message_thread_id_, so I just removed it.